resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -171,6 +173,10 @@ public boolean isWritableStackTraceEnabled() {
         return writableStackTraceEnabled;
     }
 
+    public boolean ignoreClassBindingExceptions() {
+        return ignoreUnknownExceptions;
```

Comment:
I like the name `ignoreClassBindingExceptions`
Can we also rename the variables and constants?